questionable.gemspec

```diff
@@ -7,8 +7,8 @@ require "questionable/version"
 Gem::Specification.new do |s|
   s.name        = "questionable_surveys"
   s.version     = Questionable::VERSION
-  s.authors     = ["Nick Urban, Dorian Marié"]
-  s.email       = ["nick@nickurban.com, dorian@bespokepost.com"]
+  s.authors     = ["Nick Urban, Dorian Marie, Jared Hales"]
```

Comment:
Interesting, most probably some `<= 1.8.7` ruby env, or windows. The bug is in the environment not in this file.